rst_files/ifp.rst

```diff
@@ -48,18 +48,11 @@ This method turns out to be
 
 * More efficient numerically than value function iteration
 
-
 References
 ----------------
 
 Other useful references include :cite:`Deaton1991`, :cite:`DenHaan2010`, :cite:`Kuhn2013`, :cite:`Rabault2002`,  :cite:`Reiter2009`  and :cite:`SchechtmanEscudero1977`
 
```

Comment:
@Nosferican I think the convention is to put the setup right at the end of the overview section, as opposed to before the first code block. But I don't have strong feelings about it.

Reply:
It does make working the lectures a pain. In the Setup discussion I had indicated to have them right before the `using Test` (right before any code execution). The Setup is only relevant to executing code, why should it be positioned between a bunch of markdown cells and actual code? It also forces to go up to the top to run that one cell before being able to work with the code.

Reply:
Yeah, I'm fine with either. Let's see what @jlperla thinks.